docs/tutorials/remote_client.md

```diff
@@ -1,88 +1,111 @@
-you will run on the remote node later in the instructions.
+You will need to enable linger for this user in order for the socket to work when the user is not logged in.
 
-## Setting up the remote and Podman nodes
+```
+$ sudo loginctl enable-linger $USER
+```
 
-To use the remote-client, you must perform some setup on both the remote and Podman nodes. In this case,
-the remote node refers to where the remote-client is being run; and the Podman node refers to where
-Podman and its storage reside.
+You can verify that the socket is listening with a simple Podman command.
 
+```
+$ podman --remote info
+host:
+  arch: amd64
+  buildahVersion: 1.16.0-dev
+  cgroupVersion: v2
+  conmon:
+	package: conmon-2.0.19-1.fc32.x86_64
+```
 
-### Podman node setup
+#### Enable sshd
 
-Varlink bridge support is provided by the varlink cli command and installed using:
+In order for the client to communicate with the server you need to enable and start the SSH daemon on your Linux machine, if it is not currently enabled.
 ```
-$ sudo dnf install varlink-cli
+$ sudo systemctl enable -s sshd
 ```
 
-The Podman node must have Podman (not the remote-client) installed as normal. If your system uses systemd,
-then simply start the Podman varlink socket.
+#### Setting up SSH
+Remote podman uses SSH to communicate between the client and server. The remote client works considerably smoother using SSH keys. To set up your ssh connection, you need to generate an ssh key pair from your client machine. 
 ```
-$ sudo systemctl start io.podman.socket
+$ ssh-keygen
 ```
+Your public key by default should be in your home directory under ~/.ssh/id_rsa.pub. You then need to copy the contents of id_rsa.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id.
+
+If you do not wish to use SSH keys, you will be prompted with each Podman command for your login password.  
+
+## Using the client
+
+Note: `podman-remote` is equivalent to `podman --remote` here, depending on what you have chosen to install
+
+The first step in using the Podman remote client is to configure a connection.
+
+You can add a connection by using the ‘podman-remote system connection add’ command.
 
-If your system cannot use systemd, then you can manually establish the varlink socket with the Podman
-command:
 ```
-$ sudo podman --log-level debug varlink --timeout 0  unix://run/podman/io.podman
+$ podman-remote system connection add myuser --identity ~/.ssh/id_rsa ssh://192.168.122.1/run/user/1000/podman/podman.sock
 ```
 
-### Required permissions
-For now, the remote-client requires that you be able to run a privileged Podman and have privileged ssh
-access to the remote system.  This limitation is being worked on.
+This will add a remote connection to Podman and if it is the first connection added, it will mark the connection as the default.  You can observe your connections with ‘podman system connection list’
```

Comment:
```suggestion
This will add a remote connection to Podman and if it is the first connection added, it will mark the connection as the default.  You can observe your connections with ‘podman system connection list’:
```